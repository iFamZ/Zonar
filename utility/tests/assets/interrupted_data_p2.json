{
    "type": "interrupted data part 2 - starts with longitude - matches part 1 latitude",
    "data": [
        {
            "ts": 1650500052,
            "k": "longitude",
            "v": "-122.335145"
        },
        {
            "ts": 1650500053,
            "k": "longitude",
            "v": "-122.335167"
        },
        {
            "ts": 1650500053,
            "k": "speed",
            "v": "60.45"
        },
        {
            "ts": 1650500055,
            "k": "latitude",
            "v": "47.608013"
        },
        {
            "ts": 1650500055,
            "k": "longitude",
            "v": "-122.335160"
        },
        {
            "ts": 1650500052,
            "k": "latitude",
            "v": "47.608049"
        }
    ]
}