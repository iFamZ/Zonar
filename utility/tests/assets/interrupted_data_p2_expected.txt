{'latitude': '47.608013', 'longitude': '-122.335167', 'speed': None, 'ts': 1650500050}
{'latitude': '47.608013', 'longitude': '-122.335160', 'speed': '55.15', 'ts': 1650500051}
{'latitude': '47.608049', 'longitude': '-122.335145', 'speed': '55.15', 'ts': 1650500052}
{'latitude': '47.608013', 'longitude': '-122.335160', 'speed': '60.45', 'ts': 1650500055}
Out of order detected
